index.html

```diff
@@ -4805,8 +4805,62 @@ <h2>
 layers, and pad messages to standard lengths.
       </p>
     </section>
-  </section>
 
+    <section>
+      <h2>
+          Service Privacy
+      </h2>
+      <p>
+The ability for a controller to optionally state at least one service endpoint
+in the DID document increases their control and agency. Stating more than one
+endpoint in the DID document always adds privacy risk either due to correlation
+across the endpoint descriptions or because the services are not protected by
+an authorization mechanism, or both.
+      </p>
+      <p>
+The degree of added privacy risk for using multiple service endpoints in one
+DID document can be difficult to estimate. Privacy harms are typically due to
+unintended consequences. DIDs can identify documents, things, and schemas as
+well as services. As such, they will be associated with individual people,
```

Comment:
```suggestion
unintended consequences. DIDs can identify documents, services, schemas, and
other things. As such, they will be associated with individual people,
```

Reply:
@agropper -- I've edited the above suggested change.  Perhaps it now communicates what you intended?  (My concern is that your phrasing suggests that DIDs are [or will be] used primarily to identify services, which I think inaccurate.)

Reply:
I am guessing that Adrian will approve, so I am going to update the PR with this suggestion. If I'm wrong, I'll back the change out.